### ISSUE TITLE
CONTRIBUTING.md / File ref not found

### ISSUE THREAD

**Issue by augustin-colesnic** [ezra-bible-app/ezra-bible-app#310]
@tobias-klein There's a file for emojis mentioned in the Contributing.md guide that cannot be found on the project.
<img width="1735" alt="Screenshot 2021-05-23 at 11 46 14" src="https://user-images.githubusercontent.com/592416/119254039-c64b4700-bbbc-11eb-94c0-01b3d90601d6.png">

**Comment by tobias-klein** [comment]
I think this is referring to the Android keyboard resources linked in the previous sentence, right @zhuiks? Maybe we can rephrase this to make it more clear.

**Comment by tobias-klein** [comment]
@augustin-colesnic Can you check whether the update in CONTRIBUTING.md makes things more clear? If so, we can close this issue.

**Comment by augustin-colesnic** [comment]
@zhuiks  If so, I would suggest you change the subtile 'Translating emoji categories' in '**Translating emoji categories for Android**' to be more obvious. :)

**Comment by zhuiks** [comment]
> @zhuiks If so, I would suggest you change the subtile 'Translating emoji categories' in '**Translating emoji categories for Android**' to be more obvious. :)

Well, it's more "Translating emoji categories using Android keyboard as an example". 
Emoji picker is needed only for Windows/Linux/MacOS desktops. With Cordova/Android we use native emoji picker that should be available with default screen keyboard.

Maybe we should remove the paragraph about Android keyboard completely. It's more confusing than helpful. And usually people can figure out how to translate those categories.

**Comment by augustin-colesnic** [comment]
Agree with the last sentence :) the paragraph about Android keyboard should be removed
